nncf/tensorflow/pruning/tensor_processor.py

```diff
@@ -50,3 +50,10 @@ def repeat(cls, tensor: NNCFTensor, repeats: int) -> NNCFTensor:
     def elementwise_mask_propagation(cls, input_masks: List[NNCFTensor]) -> NNCFTensor:
         cls.assert_allclose(input_masks)
         return input_masks[0]
+
+    @classmethod
+    def split(cls, tensor: NNCFTensor, chunks: int, axis: int, output_shapes: List[int] = None) -> List[NNCFTensor]:
+        if axis >= len(tensor.tensor.shape):
+            axis = -1
```

Comment:
When this happens? I preferer to have an exception instead of this if

Reply:
This happened frequently. We usually set axis as -1 but during `get_split_axis`, it could be considered like axis=3. And in this case, there could be occurred the error. So I put this if clause to prevent this case.

Reply:
Could you please present an example?

Reply:
For example, in the `SplitConcatModel` or `MultipleSplitConcatModel` ,  the models of `tests/torch/pruning.test_model_pruning_analysis.py`.  In this case, the axis is set as `1`, but the `len(tensor.tensor.shape)` is also 1, so the if we did not change axis as `-1`, the chunk/split operation could not be performed.

Reply:
But we know here that tensor always has only one dimension, so we don't need `axis` param here

Reply:
Let's make it similar to concat axis: `axis` param is required but always 0 is used as its value. We will use this param when NNCFTensor become more than 1D

Reply:
Remove this as you said, And I explained it in [conversation](https://github.com/openvinotoolkit/nncf/pull/1244#discussion_r949806730).